packages/space-opera/src/components/camera_settings/components/limits_base.ts

```diff
@@ -23,15 +23,17 @@ import '../../shared/checkbox/checkbox.js';
 
 import {html, query} from 'lit-element';
 
+import {snackbarStyles} from '../../../styles.css.js';
 import {ConnectedLitElement} from '../../connected_lit_element/connected_lit_element.js';
 import {SliderWithInputElement} from '../../shared/slider_with_input/slider_with_input.js';
 import {Limits} from '../types.js';
 
 import {styles} from './limits_base.css.js';
 
+
 /** Abstract component that can be extended for editing scalar limits. */
 export abstract class LimitsBase extends ConnectedLitElement {
-  static styles = styles;
+  static styles = [styles, snackbarStyles];
```

Comment:
The snackbar strikes again!

Reply:
lol: https://stackoverflow.com/questions/34432339/android-snackbar-vs-toast-usage-and-difference
tldr; Toasts are snackbars without actionable content

Reply:
so technically yes, this should be named toast

Reply:
lol, that's good to know. I'd never have guessed from their names! I guess this is how we know I'm not a UX one...